components/generic/Badge/Badge.tsx

```diff
@@ -5,10 +5,8 @@ type BadgeProps = {
 
 export const Badge = ({ textContent, color }: BadgeProps) => {
   return (
-    <span
-      className={`pt-0.5 pb-1 px-2 bg-${color}-100 text-${color}-800 text-xs leading-4 font-normal rounded-3xl`}
-    >
-      {textContent}
-    </span>
+    <div className={`flex pb-0.5 px-2 bg-${color}-100 rounded-3xl`}>
+      <span className={`text-${color}-800 text-xs leading-4 font-normal`}>{textContent}</span>
```

Comment:
Error, prone. I don't think tailwind exposes types, but it could, can we use that props?

Reply:
What do you mean here exactly?